harvester_e2e_tests/apis/test_images.py

```diff
@@ -18,6 +18,7 @@
 
 from time import sleep
 from datetime import datetime, timedelta
+from pytest_cases import parametrize_with_cases, case
```

Comment:
3rd party library should be placed in 2nd section. (PEP8)

Reply:
Updated :smile: :+1: